dist/common/scripts/scylla_raid_setup

```diff
@@ -37,14 +37,14 @@ if __name__ == '__main__':
                         help='specify disks for RAID')
     parser.add_argument('--raiddev', default='/dev/md0',
                         help='MD device name for RAID')
-    parser.add_argument('--update-fstab', action='store_true', default=False,
-                        help='update /etc/fstab for RAID')
     parser.add_argument('--root', default='/var/lib/scylla',
                         help='specify the root of the tree')
     parser.add_argument('--volume-role', default='all',
                         help='specify how will this device be used (data, commitlog, or all)')
     parser.add_argument('--force-raid', action='store_true', default=False,
                         help='force constructing RAID when only one disk is specified')
+    parser.add_argument('--disable-on-nextboot', action='store_true', default=False,
+                        help='do not mount RAID on next boot')
 
```

Comment:
Let's make it `--enable-on-nextboot` (default true) to avoid double negatives. And make `--update-fstab` an alias to avoid breaking scripts.